rose/server/game.py

```diff
@@ -1,12 +1,15 @@
 import random
 import operator
 from twisted.internet import reactor, task
+import logging
```

Comment:
Can you move the logging import before the twisted import?

We like to organize imports from less specific to more specific:
1. stdlib imports
1. 3rd party imports
1. rose imports
1. this package imports